nimbus.fml.yaml

```diff
@@ -307,6 +307,9 @@ features:
             USER_FR_SPEAKER:          "'fr' in locale"
             DEVICE_ANDROID:           os == 'Android'
             DEVICE_IOS:               os == 'iOS'
+            I_AM_DEFAULT_BROWSER: "is_default_browser"
```

Comment:
Nit: whitespace. It makes it easier to read the YAML if the expressions all line up.